### DISCUSSION TITLE
/mirai login 指令无法正常使用

### DISCUSSION THREAD

**Discussion by Gertvy** [DreamVoid/MiraiMC#166] [问答 / Q&A]
在今日早上我开启服务器时，发现原本已经搞好的机器人不工作了。我想要重新登录机器人账号，并在控制台输入了mirai login 账号 密码 但是却提示我输入mirai help 并且输入mirai help 也输出和前者一样的信息。
[10:17:47 INFO]: For help, type /mirai help

**Comment by DreamVoid** [comment]
然后你想表达什么，你想让我解决什么

**Reply by Gertvy** [reply]
> 然后你想表达什么，你想让我解决什么
如何解决

**Reply by suxingye** [reply]
Error occurred while disabling MiraiMC v1.7-pre2 (Is it up to date?) java.lang.NullPointerException: Cannot invoke "net.mamoe.mirai.event.Listener.complete()" because "this.BotOnlineListener" is null
昨天至现在我也遇到了同样的问题 插件无法工作 我尝试了多台服务器均无法使用 使用这个前置开发的其他插件的交流群内的很多服主都遇到了这个问题 测试的服务端purpur1.19 paper1.19

**Reply by DreamVoid** [reply]
确定是问题就去开issue
